backend/drive/drive_loadbalance.go

```diff
@@ -0,0 +1,304 @@
+package drive
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"math/rand"
+	"net/http"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/valyala/fastjson"
+	"golang.org/x/oauth2/google"
+	"golang.org/x/oauth2/jwt"
+	"gopkg.in/dc0d/tinykv.v4"
```

Comment:
I note that the underlying github for this has been set to read only and archived

https://github.com/workshop-depot/tinykv

Can we use rclone's `lib/cache` instead maybe?